app/assets/stylesheets/finder_frontend.scss

```diff
@@ -1,3 +1,5 @@
+$colour-true-black: #000;
```

Comment:
can we prefix this with `app-` to make sure it doesn't clash with who-knows-what?

Reply:
:) done